File: BTLmnm/app/src/main/java/com/example/mypc/btlmnm/entries/TaiKhoan.java
package com.example.mypc.btlmnm.entries;

import java.util.ArrayList;

/**
 * Created by thandieuhaui on 09-Aug-17.
 */

public abstract   class TaiKhoan {
    private String idUser = "";
    private String userName = "";
    private String matKhau = "";
    private String hoDem = "";
    private String ten ="";
    private String ngaySinh ="";
    private String queQuan="";
    private String quyen = "";

    public TaiKhoan() {
    }


    public TaiKhoan(String idUser, String userName, String matKhau, String hoDem, String ten, String ngaySinh, String queQuan, String quyen) {
        this.idUser = idUser;
        this.userName = userName;
        this.matKhau = matKhau;
        this.hoDem = hoDem;
        this.ten = ten;
        this.ngaySinh = ngaySinh;
        this.queQuan = queQuan;
        this.quyen = quyen;
    }

    public String getIdUser() {
        return idUser;
    }

    public void setIdUser(String idUser) {
        this.idUser = idUser;
    }

    public String getUserName() {
        return userName;
    }

    public void setUserName(String userName) {
        this.userName = userName;
    }

    public String getMatKhau() {
        return matKhau;
    }

    public void setMatKhau(String matKhau) {
        this.matKhau = matKhau;
    }

    public String getHoDem() {
        return hoDem;
    }

    public void setHoDem(String hoDem) {
        this.hoDem = hoDem;
    }

    public String getTen() {
        return ten;
    }

    public void setTen(String ten) {
        this.ten = ten;
    }

    public String getNgaySinh() {
        return ngaySinh;
    }

    public void setNgaySinh(String ngaySinh) {
        this.ngaySinh = ngaySinh;
    }

    public String getQueQuan() {
        return queQuan;
    }

    public void setQueQuan(String queQuan) {
        this.queQuan = queQuan;
    }

    public String getQuyen() {
        return quyen;
    }

    public void setQuyen(String quyen) {
        this.quyen = quyen;
    }

    @Override
    public String toString() {
        return "TaiKhoan{" +
                "idUser='" + idUser + '\'' +
                ", userName='" + userName + '\'' +
                ", matKhau='" + matKhau + '\'' +
                ", hoDem='" + hoDem + '\'' +
                ", ten='" + ten + '\'' +
                ", ngaySinh='" + ngaySinh + '\'' +
                ", queQuan='" + queQuan + '\'' +
                ", quyen='" + quyen + '\'' +
                '}';
    }
}

File: mnm/model/Taikhoan.php
<?php 
/**
* 
*/
class TaiKhoan
{
	
	private $iD;
	private $userName;
	private $passWord;
	private $ho;
	private $ten;
	private $ngaySinh;
	private $queQuan;
	private $quyen;

	function __construct()
	{

	}
	function setId($id)
	{
		$this->iD=$id;
	}
	function setUserName($username)
	{
		$this->userName=$username;
	}

	function setPassWord($password)
	{
		$this->passWord=$password;
	}
	function setHo($ho)
	{
		$this->ho=$ho;
	}

	function setTen($ten)
	{
		$this->ten=$ten;
	}
	function setNgaySinh($ngaysinh)
	{
		$this->ngaySinh=$ngaysinh;
	}
	function setQueQuan($quequan)
	{
		$this->queQuan=$quequan;
	}
	function setQuyen($quyen)
	{
		$this->quyen=$quyen;
	}

  function getId()
	{
		return $this->iD;
	}
	function getUserName()
	{
		return $this->userName;
	}

	function getPassWord()
	{
		return $this->passWord;
	}
	function getHo()
	{
		return $this->ho;
	}

	function getTen()
	{
		return $this->ten;
	}
	function getNgaySinh()
	{
		return $this->ngaySinh;
	}
	function getQueQuan()
	{
		return $this->queQuan;
	}
	function getQuyen()
	{
		return $this->quyen;
	}

	function Login($username,$password)
	{
		require_once "model/Connect.php";
		$kn=new Connect();
		$conn=$kn->KetNoi();
		$result=$conn->query("select * from taikhoan where username='".$username."' and matkhau='".$password."'");
		$tk=new TaiKhoan();
		if ($result->num_rows > 0) {
			while ($row=$result->fetch_array()) {
				$tk->iD=$row['id_user'];
				$tk->userName=$row['username'];
				$tk->passWord=$row['matkhau'];
				$tk->ho=$row['hodem'];
				$tk->ten=$row['ten'];
				$tk->ngaySinh=$row['ngaysinh'];
				$tk->queQuan=$row['quequan'];
				$tk->quyen=$row['quyen'];
			}
		}
		$kn->DisConnect();
		return $tk;
	}

	/*function getTk($id)
	{
		require_once "model/Connect.php";
		$kn=new Connect();
		$conn=$kn->KetNoi();
		$result=$conn->query("select * from taikhoan where id_user=".$id);
		$tk=new TaiKhoan();
		if ($result->num_rows > 0) {
			while ($row=$result->fetch_array()) {
				$tk->iD=$row['id_user'];
				$tk->userName=$row['username'];
				$tk->passWord=$row['matkhau'];
				$tk->ho=$row['hodem'];
				$tk->ten=$row['ten'];
				$tk->ngaySinh=$row['ngaysinh'];
				$tk->queQuan=$row['quequan'];
				$tk->quyen=$row['quyen'];
			}
		}
		$kn->DisConnect();
		return $tk;
	}*/

	function DoiPass($idUser,$username,$password,$passmoi)
	{
		require_once "model/Connect.php";
		$kn=new Connect();
		$conn=$kn->KetNoi();
		$result=$conn->query("select * from taikhoan where username='".$username."' and matkhau='".$password."'");
		$tb="";
		if ($result->num_rows > 0) {
			$kq=$conn->query("update taikhoan set matkhau='".$passmoi."' where 
				id_user=".$idUser);
			if ($kq===true) {
				$tb="1";
			}
			else
			{
				$tb="0";
			}
		}
		$kn->DisConnect();
		return $tb;
	}



}
 ?>